packages/npm-dependencies-validation/test/projects/no_deps_installed/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "no_deps_installed",
+  "version": "1.0.0",
+  "dependencies": {
+    "lodash": "~4.17.21",
```

Comment:
use fixed version as it may change test outputs in future if a different version is installed

Reply:
Nothing is installed in tests.
I want to check that all types of versions are returned correctly